low-level/framework/utils/utility.py

```diff
@@ -58,3 +60,55 @@ def is_env_vm(self):
             logger.warning("Error while reading whether env is vm or not, assuming VM env : {e}")
         return is_vm
 
+    @staticmethod
+    def _send_command(command: str, fail_on_error=True):
```

Comment:
If the method is intended to be used by other modules, then it should be a public method. If we use it in the same module then it can be a private method.

Reply:
Done